### PR TITLE
[API][MyTeam] Initialize MyTeam

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: features/api/my_teams.feature [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    When I send a "POST" request to "/api/v3/my_teams?scope=referent"
+    Then the response status code should be 201
+    And the JSON should be equal to:
+    """
+    {
+        "uuid": "@uuid@"
+    }
+    """
+
+  Scenario: As a referent I will get my team, when I want create it and it exists
```
Comment (elvino29) [inline features/api/my_teams.feature:44]:
```suggestion
  Scenario: As a referent I will get my team When i want to create it but it already exists
```

File: src/Controller/Api/MyTeam/InitializeMyTeamController.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+class InitializeMyTeamController extends AbstractController
+{
+    public function __invoke(
+        Request $request,
+        ScopeGeneratorResolver $scopeGeneratorResolver,
+        MyTeamRepository $myTeamRepository,
+        EntityManagerInterface $entityManager
+    ): ?MyTeam {
+        if (!$scopeCode = $request->query->get('scope')) {
+            throw new BadRequestHttpException('Pas de scope.');
```
Comment (elvino29) [inline src/Controller/Api/MyTeam/InitializeMyTeamController.php:22]:
```suggestion
            throw new BadRequestHttpException('Aucun scope renseigné.');
```

File: src/Controller/Api/MyTeam/InitializeMyTeamController.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        }
+
+        $scope = $scopeGeneratorResolver->generate();
+        if (!$scope) {
+            throw new BadRequestHttpException('L\'utilisateur n\'a pas de scope demandé.');
```
Comment (elvino29) [inline src/Controller/Api/MyTeam/InitializeMyTeamController.php:27]:
```suggestion
            throw new BadRequestHttpException('Vous n\'avez pas accès au scope demandé.');
```

**Review by elvino29 — APPROVED** [review]

File: src/Controller/Api/MyTeam/InitializeMyTeamController.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        Request $request,
+        ScopeGeneratorResolver $scopeGeneratorResolver,
+        MyTeamRepository $myTeamRepository,
+        EntityManagerInterface $entityManager
+    ): ?MyTeam {
+        if (!$scopeCode = $request->query->get('scope')) {
+            throw new BadRequestHttpException('Aucun scope renseigné.');
+        }
+
+        $scope = $scopeGeneratorResolver->generate();
```
Comment (Remg) [inline src/Controller/Api/MyTeam/InitializeMyTeamController.php:25]: you should use the `resolve` method instead

**Review by Remg — APPROVED** [review]

**Pull request merged**